sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -34,7 +34,9 @@ public partial class AzureKeyCredential
         public override int GetHashCode() { throw null; }
         public static bool operator ==(Azure.ETag left, Azure.ETag right) { throw null; }
         public static bool operator !=(Azure.ETag left, Azure.ETag right) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Why do you hide this one?

Reply:
Initially the thought was that `ToFormattedString` would only implement the header format behavior which would be the new supported default to move our code, including generated code, away from handling the string formatting concern of adding quote wrapping. Subsequently we planned to ban the use of the current `ToString()` method, via analyzer, to prevent future use in our code.

Now that we moved to this overload the guidance is less clear. as callers can still request the old default format behavior. Do you have any suggestions on how we can still enforce this new formatting behavior given the current methods?

Reply:
I think using the old inconsistent behavior would result in a bug most of the time. we are keeping it for back compact only.

Reply:
I think it's very strange to see parameterized ToString but not default ToString.

Reply:
Also, it seems like many callsites below use ToString.

Reply:
The plan was to move all callsites we control to the formatted overload once we update generation to do the same.